common/defs/support.json

```diff
@@ -189,9 +188,8 @@
       "eth:ETC:61": "2.6.1",
       "eth:ETH:1": "2.6.1",
       "eth:MATIC:137": "2.6.1",
-      "eth:tETH:3": "2.6.1",
-      "eth:tETH:4": "2.6.1",
-      "eth:tETH:5": "2.6.1",
+      "eth:tSEP:11155111": "2.6.4",
+      "eth:tHOL:17000": "2.6.4",
```

Comment:
you just added `tSEP` and `tHOL` in this PR. the correct support version is 2.8.2 (next core release), dtto below